pkg/ingester/series.go

```diff
@@ -23,7 +23,7 @@ func init() {
 }
 
 type memorySeries struct {
-	metric model.Metric
+	metric labelPairs
```

Comment:
perhaps this should be called `labels` now?

Reply:
I left most names the same to avoid creating billions of diffs that don't really change anything.
Once you're otherwise happy with it I can change that.